subsys/net/lib/wifi_provision/proto/common.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+enum Band {
+  BAND_UNSPECIFIED = 0;
+  BAND_2_4_GHZ = 1;
+  BAND_5_GHZ = 2;
+  BAND_6_GHZ = 3;
+}
+
+enum AuthMode {
+  OPEN = 0;
+  WEP = 1;
+  WPA_PSK = 2;
+  WPA2_PSK = 3;
+  WPA_WPA2_PSK = 4;
+  WPA2_ENTERPRISE = 5;
+  WPA3_PSK = 6;
+}
+
+message WifiInfo {
+  string ssid = 1;
+  Band band = 3;
+  uint32 channel = 4;
+  AuthMode authMode = 5;
+}
+
+message WifiConfig {
+  WifiInfo info = 1;
+  string passphrase = 2;
```

Comment:
This passphrase must be nullable in the case of an open network.

Reply:
Im in the process of updating the schema to be _identical_ to the BLE counterpart :+1: